experimental/dds/tree2/src/core/tree/anchorSet.ts

```diff
@@ -410,56 +408,28 @@ export class AnchorSet implements ISubscribable<AnchorSetRootEvents> {
 	}
 
 	/**
-	 * Updates paths for a range move (including re-parenting path items and updating indexes).
-	 * @param count - number of siblings to insert/delete/move.
-	 * @param srcStart - where the siblings are removed from. If undefined the operation is an insert.
-	 * @param dst - where the siblings are moved to. If undefined the operation is a delete.
-	 *
-	 * TODO:
-	 * How should anchors that become invalid, then valid again (ex: into content that was deleted, then undone) work?
-	 * Add an API to resurrect them? Store them in special detached fields? Store them in special non-detached fields?
-	 *
-	 * TODO:
-	 * How should custom anchors work (ex: ones not just tied to a specific Node)?
-	 * This design assumes they can be expressed in terms of a Node anchor + some extra stuff,
-	 * but we don't have an API for the extra stuff yet.
+	 * Detach nodes from their parent.
```

Comment:
If you keep this method, it should really clarify that the nodes are not placed in a detached sequence, and that they must be inserted into the tree somewhere else before any other operations (such as removal of events or decrementing of ref counts) occurs in any of the returned subtrees of anchors. Also it should clarify what the indexes and parents of these nodes are while they are detached.